### DISCUSSION TITLE
Show species info?

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#15852] [Q&A] [answered]
Is there some way to see species info on the map in OsmAnd? Specifically I was hoping to see when trees in orchards and arboreta have info in the species (or genus) tag.

As far as I can tell it is only possible to tap and select trees on the map that use the name tag and even those don't show additional species info in details.

**Comment by vshcherb** [comment]
Some of that is rendered by default style but we don't expect to have all variety.

**Comment by xmd5a2** [comment, marked as answer]
We do support [`species`](https://wiki.openstreetmap.org/wiki/Key%3Aspecies) tag but only as additional text info. It is not possible to filter such POIs. Of course if OSM data for particular object doesn't provide this tag then nothing will be shown.
